assets/railcraft/lang/pl_pl.lang

```diff
@@ -83,10 +83,17 @@ death.railcraft.bore.5=%s utkwił pomiędzy skałą a twardym miejscem
 
 death.railcraft.train.1=%s został rozjechany 
 death.railcraft.train.2=%s bawił się na torach
-death.railcraft.train.3=%s bawil się w kurczka z pociągiem; pociąg wygrał
+death.railcraft.train.3=%s bawił się w kurczka z pociągiem; pociąg wygrał
 death.railcraft.train.4=%s złapał nie ten pociąg
 death.railcraft.train.5=%s kupił bilet w jedną stronę do zaświatów
 
+death.railcraft.electric.1=%s wita się z Nicolą Tesla
+death.railcraft.electric.2=%s odkrył znaczenie "Uwaga, wysokie napięcie!"
+death.railcraft.electric.3=%s został olśniony przez cuda elektryczności
```

Comment:
Dałbym `oświecony`. Może też `potęgę` zamiast `cuda` - deathmesgi mają być śmieszne, więc lepsze będą metody lokalizacyjne i szukanie polskich odpowiedników angielskich sformułowań!

Reply:
Oświecony to słówko którego szukałem 😊

Reply:
Dodatkowo, gdzieś w polskim tłumaczeniu bewitched było napisane „padało psami i kotami” 😂 To odnośnie tych tłumaczeń. Ale dzięki!